demos/git/README.md

```diff
@@ -0,0 +1,27 @@
+This example builds git trunk. It demonstrates the use of the 'home'
+plug to allow it access to the users home directory.
+
+After building and installing the snap, you will need to run the following
+command to grant it access to your home directory:
+
+```sh
+snap connect git:home ubuntu-core:home
+```
+
+Only access the home directory is granted. You can see this containment
```

Comment:
*Only access TO the home ...
I think we should explain the type of access. From snapd's docs: Can access non-hidden files in user's `$HOME` to read/write/lock.